src/components/Slices/ConfirmAddress.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useReducer, useState } from "react";
```

Comment:
Most of this code came from the AddressExpander component. There was a lot going on there that didn't need to all be together. Moving this here gives us some separation of concerns and allows us to disconnect this from the redux store. 

The reducer here is just a hook and is only operating on local state. Ideally this makes the component more flexible for use in other applications.